### PR TITLE
feat(versions): Explain top-level versions

### PR DESCRIPTION
**Pull request by lwander**
@spinnaker/google-reviewers

### CODE REVIEW

**Review by skim1420 — APPROVED** [review]

File: community/releases/versions.md [hunk 1]
```diff
@@ -5,6 +5,22 @@ sidebar:
   nav: community
 ---
 
+The Spinnaker releases listed below are top-level versions tying together each
+Spinnaker subcomponents' versions. These have been validated together in an 
+end-to-end integration test suite curated by the Spinnaker community, and
+represent a more maintainable, easy to upgrade Spinnaker. While 
+it's still possible to install each Spinnaker subcomponent's versions
```
Comment (ttomsu) [inline community/releases/versions.md:12]: Link for those wanting to learn how to modify the component versions?

**Review by ttomsu — APPROVED** [review]

File: community/releases/versions.md (hunk 1 above)
Comment (lwander) [reply community/releases/versions.md:12]: What do you mean? Publish your own BOM?

Comment (ttomsu) [reply community/releases/versions.md:12]: Right - is there documentation on that? If so, I'd link it here.

Comment (lwander) [reply community/releases/versions.md:12]: There is not - and I've only seen it asked about once by a user with very unique constraints. It's kind of an advanced topic and this doesn't feel like the right place to bring it up.

Comment (lwander) [reply community/releases/versions.md:12]: We should (at some point) publish comprehensive release docs that address this.

Comment (ttomsu) [reply community/releases/versions.md:12]: I would then remove the mention that it's possible, and let users ask for it when appropriate.

**Comment by lwander** [comment]
https://github.com/spinnaker/spinnaker/issues/1742

File: community/releases/versions.md (hunk 1 above)
Comment (lwander) [reply community/releases/versions.md:12]: Sorry merged before I saw your comment. 

The mention alludes to direct downloads (which users are currently doing).

Comment (lwander) [reply community/releases/versions.md:12]: The point is for this to contrast the existing install path (downloading the packages via whatever path users are going by) against the Halyard install path. It's not explaining that Halyard can allow you to download whatever versions you want.

**Pull request merged**